src/test/java/org/ilgcc/app/journeys/ActivitiesJourneyTest.java

```diff
@@ -0,0 +1,550 @@
+        testPage.navigateToFlowScreen("gcc/activities-parent-intro");
+        saveSubmission(getSessionSubmissionTestBuilder().withDayCareProvider()
+            .withParentDetails()
+            .withChild("First", "Child")
+            .withChild("Second", "Child")
+            .build());
+
+        //activities-parent-intro
+        assertThat(testPage.getTitle()).isEqualTo("Activities Parent Intro");
+        testPage.clickContinue();
+        //activities-parent-type
+        assertThat(testPage.getTitle()).isEqualTo("Activities Parent Type");
+        testPage.clickElementById("activitiesParentChildcareReason-SCHOOL");
+        testPage.clickContinue();
+
+        //activities-add-ed-program
+        assertThat(testPage.getHeader()).isEqualTo("Tell us about your school or training program.");
+        testPage.clickContinue();
+
+        //activities-ed-program-type
+        assertThat(testPage.getElementText("educationType-highSchool-label")).isEqualTo("High School or GED");
+        testPage.clickElementById("educationType-highSchool-label");
+        assertThat(testPage.getHeader()).isEqualTo("What type of school or training are you enrolled in?");
+        testPage.clickContinue();
+
+        //activities-ed-program-name
+        testPage.enter("schoolName", "World");
+        assertThat(testPage.getHeader()).isEqualTo("What is the school or training program name?*");
+        testPage.clickContinue();
+
+        //activities-ed-program-info
+        assertThat(testPage.getTitle()).isEqualTo("School or training program");
+        testPage.clickContinue();
+
+        //activities-ed-program-method
+        assertThat(testPage.getTitle()).isEqualTo("Learning style");
+        testPage.clickElementById("programTaught-Online-label");
+        testPage.clickElementById("programSchedule-No-label");
+        testPage.clickContinue();
+
+        //activities-next-class-schedule
+        assertThat(testPage.getHeader()).isEqualTo("Next, we'll ask about your class schedule.");
+        testPage.clickContinue();
+
+        //activities-class-weekly-schedule
+        assertThat(testPage.getTitle()).isEqualTo("Weekly Class Schedule");
+        testPage.clickElementById("weeklySchedule-Monday");
+        testPage.clickElementById("weeklySchedule-Sunday");
+        testPage.clickContinue();
+
+        //activities-class-hourly-schedule
+        assertThat(testPage.getTitle()).isEqualTo("Hourly Class Schedule");
+        testPage.clickElementById("activitiesClassHoursSameEveryDay-Yes");
+        testPage.enter("activitiesClassStartTimeAllDays", "0900AM");
+        testPage.enter("activitiesClassEndTimeAllDays", "13:00");
+        testPage.clickContinue();
+
+        //activities-ed-program-dates
+        assertThat(testPage.getTitle()).isEqualTo("Time of Program");
+        testPage.clickContinue();
+
+        assertThat(testPage.getTitle()).isEqualTo("Unearned Income Intro");
+    }
+
+    @Test
+    void ParentOnlyWithNoJobOrWorkTest() {
```

Comment:
I think there's a typo in the method name:

```suggestion
    void ParentOnlyWithNoJobOrSchoolTest() {
```

Reply:
Love this!